tests/e2e/tests/pilot/egress_rules_test.go

```diff
@@ -23,6 +23,8 @@ import (
 
 	multierror "github.com/hashicorp/go-multierror"
 
+	"testing"
```

Comment:
nit: bad import block, run goimports please.

Reply:
are the linters broken/not finding those ?